cluster/kubemark/config-default.sh

```diff
@@ -22,7 +22,7 @@
 GCLOUD=gcloud
 ZONE=${KUBE_GCE_ZONE:-us-central1-b}
 MASTER_SIZE=${MASTER_SIZE:-n1-standard-4}
-NUM_MINIONS=${NUM_MINIONS:-100}
+NUM_MINIONS=${NUM_MINIONS:-4000}
```

Comment:
Yes, that's what I mentioned to you in person.  I'd prefer to fix that in a separate PR.
It will try to create 4000 pseudo-nodes (in pods) on 10 actual node VM's, and fail.